Release_Notes.md

```diff
@@ -6,6 +6,16 @@
 
 *   Functions that are called from Protected Audience worklets are now only accessible from inside the worklets, not from the global scope.  See [#489](https://github.com/WICG/turtledove/issues/489) for more information.
 
+## Chrome M114
+
+* Support the ability to specify `requestedSize` in the auction config, which is eventually stored in the winning fenced frame config's container size. The `requestedSize` may not be accessible through browser signals in the auction until M116, and is a lower priority because it is a convenience feature only (presumably the size of the ad slot is already passed in through other signals, if it is needed).
```

Comment:
Thiiiink there might be a need for some disambiguation here. 
- The `requestedSize` is an `auctionConfig` feature known to the publisher at auction time.
- The `renderSize` is the size of the creative from the IG, and can be different than the `requestedSize`.
- `requestedSize` of the auction can get into the reporting functions through auction config and/or auction signals.
- `renderSize` of the chosen creative would need to get in through `browserSignals`.

Is the intention here to say that as of 116 the `renderSize` will be included in the k-anon check and therefore passed into `reportResult` and `reportWin` through the `browserSignals`?

Reply:
I'm also not entirely sure what it means if different component auctions have different `requestedSize` values w.r.t the top-level seller auction; in the expected PBJS workflow, the top-level auction config isn't known in advance, nor configured by the publisher; so what's the responsibility of the top-level seller in terms of handling different sizes coming back from component auctions?

Reply:
"that as of 116 the renderSize will be included in the k-anon check"

No, the size being included in the k-anon check will come later (and then yes, renderSize can be in the browser signals).

We just hadn't implemented requestedSize in browser signals until M116, no relation to k-anon.

Reply:
@rdgordon-index I'm not sure how the top-level auction config wouldn't know the size of the ad; the caller needs to know how big to make the ad, right? (and doesn't get anything returned from the auction to indicate this) This is probably related to the other issue about lack of multi-size auctions.

Reply:
I'm thinking about the expected PBJS integration -- where the sellers are responsible for the component auction configs, but GAM (via GPT) is responsible for building the top-level seller auction config thereafter; so it's the order-of-operations that makes this somewhat confusing if not all sellers (component or top-level) interpret the size array of a GPT slot in the same fashion.

Reply:
https://github.com/WICG/turtledove/issues/908 talks about this as well

Reply:
https://github.com/google/ads-privacy/tree/master/proposals/fledge-multiple-seller-testing#faq